script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -365,7 +364,7 @@ Well, even if you take that into account,
 [sha:134f70ea9a4ba4b1de8a6c8d565a77056fd294de]{
 -- Page 71, Offset 1337.
 -- 「……いや、最近の中学生はおっかないな」
-"...Man, middle school students these days sure are intimidating."
+"Man... middle school students these days sure are intimidating."
 }
 [sha:d52e59a52750dbcb21c8affe8856f4ae5a3f3612]{
 -- Page 71, Offset 1338.
```

Comment:
To give so harsh a critique while simply passing by, it's akin to a wandering samurai casually executing a peasant.

Reply:
I prefer the original phrasing for the latter half of the sentence.

> she's just like a samurai casually executing a peasant.

Reply:
Yup can keep that, I wonder if we can lose the 'just' though, it's a bit extraneous. But overall correcting the word order error is the important thing, really.

Reply:
sure, just is a bit redundant yeah